packages/inter-protocol/src/reserve/assetReserve.js

```diff
@@ -56,18 +48,11 @@ const nonalphanumeric = /[^A-Za-z0-9]/g;
 
 /**
  * Asset Reserve holds onto assets for the Inter Protocol, and can
- * dispense it for various purposes under governance control. It currently
- * supports governance decisions to add liquidity to an AMM pool.
+ * dispense it for various purposes under governance control.
```

Comment:
It can't really dispense until it's upgraded, right? Is that implicit in "under governance control"?

Reply:
Yeah. that's what I was thinking.